lib/matplotlib/tests/test_axes.py

```diff
@@ -2261,6 +2261,24 @@ def test_errorbar_limits():
     ax.set_title('Errorbar upper and lower limits')
 
 
+@image_comparison(baseline_images=['errorbar_offsets'],
+                  extensions=['png', 'svg'])
+def test_errorbar_offsets():
+    x = np.linspace(0, 1, 15)
+    y = x * (1-x)
+    yerr = y/6
+    fig = plt.figure()
+    ax1 = fig.add_subplot(2, 1, 1)
+    ax2 = fig.add_subplot(2, 1, 2)
+    for shift in [0, 0, 2, 3]:
```

Comment:
I wanted to have a repeated value to ensure that error bars overlap when they're asked to.

Sidenote:  I haven't tested using x-axis errors or different upper and lower errors.  Shouldn't make a difference, but maybe the test should check anyway.